app/web/features/search/SearchResultsList.tsx

```diff
@@ -139,11 +135,11 @@ export default function SearchResultsList({
       <Hidden smDown>
         <SearchBox
           searchType={searchType}
-          setSearchType={setSearchType}
           locationResult={locationResult}
+          queryName={queryName}
+          setSearchType={setSearchType}
           setLocationResult={setLocationResult}
           setQueryName={setQueryName}
```

Comment:
`setQueryName`, `setLocationResult`, `setSearchType` no de-coupled here, trust me if I say I want to implement some features before :D